python/cuml/fil/fil.pyx

```diff
@@ -331,7 +354,12 @@ cdef class ForestInference_impl():
 
         cdef treelite_params_t treelite_params
 
-        treelite_params.output_class = output_class
+        self.output_class = output_class
+        if not self.output_class:
```

Comment:
How do we know if a classification model is being loaded? The only way we can find it out is the `output_class` parameter.

Reply:
No need for a warning if `output_class==False`: it's fine for regression models.

Reply:
I have removed the warning statement. 
The idea behind the warning was to let users know that their output_class=False and so if they are using a classification model then they need to set it to true. This scenario is possible when FIL is directly used.